app/javascript/gobierto_data/webapp/components/commons/Info.vue

```diff
@@ -155,6 +148,12 @@ export default {
     },
     checkStringLength() {
       return this.descriptionDataset.length > 250
+    },
+    hasSourceDataset() {
```

Comment:
the second part of the condition should check url attribute?

Reply:
not here, we are only interested into have a not empty text ... the checks about condition url are into InfoBlockText.vue

    hasSourceUrl() {
      return this.source.url !== undefined && this.source.url !== ""
    }

Reply:
ok!